HPC/README.md

```diff
@@ -0,0 +1,117 @@
+# Getting started with HPC workloads on Graviton instances
+## Contents
+* [Introduction](#introduction)
+* [Compilers](#compilers)
+* [Libraries](#libraries)
+* [EFA](#efa)
+* [OpenMPI](#openmpi)
+* [Parallel Cluster](#parallel-cluster)
+* [Examples and Benchmarks](#examples-and-benchmarks)
+
+## Introduction
+The performance of HPC applications heavily depends on compute, network and IO. The Graviton3E based HPC7g instance has the right configuration to deliver the best performance for multi-instance cluster HPC applications: 35% higher vector bandwidth with no power throttling, 200 Gbps of network bandwidth. For several widely used HPC applications on cloud, we are able to achieve comparable performances and lower costs than x86 instances specialized for HPC. This document provides a walkthrough on how to compile HPC softwares and obtain the best performance on Graviton EC2 instances. 
```

Comment:
with low core host counts we'll see better perf, i don't thin you want to say comparable perf